cms-api/src/main/java/com/commercetools/sunrise/cms/CmsService.java

```diff
@@ -6,16 +6,20 @@
 import java.util.concurrent.CompletionStage;
 
 /**
- * Service that provides page content, coming from some sort of Content Management System (CMS).
+ * A service providing access to CMS pages.
  */
-@FunctionalInterface
```

Comment:
I would suggest to keep the annotation here because this interface is a SAM (single abstract method) type.